Add '@babel/core'

With my pnpm setup secondar dependencies are not solved automatically. Therefore I get:
```
[!] Error: Cannot find module '@babel/core'
```

An `nmp i -D @babel/core` solved the issue. Yoy might want to add the dependency to your builder as well? Thanks.

--- a/templates/extension-js/package.json
+++ b/templates/extension-js/package.json
@@ -13,6 +13,7 @@
     "dev": "npm run clean && rollup -c -w"
   },
   "devDependencies": {
+    "@babel/core": "^7.21.0",
     "@rollup/plugin-babel": "^6.0.3",
     "@rollup/plugin-commonjs": "^24.0.1",
     "@tiptap/core": "^2.0.0-beta.220",

--- a/templates/extension-ts/package.json
+++ b/templates/extension-ts/package.json
@@ -11,6 +11,7 @@
     "dev": "npm run clean && rollup -c -w"
   },
   "devDependencies": {
+    "@babel/core": "^7.21.0",
     "@rollup/plugin-babel": "^6.0.3",
     "@rollup/plugin-commonjs": "^24.0.1",
     "@tiptap/core": "^2.0.0-beta.220",

